frame/staking-rewards/src/lib.rs

```diff
@@ -611,6 +614,28 @@ pub mod pallet {
 		}
 	}
 
+	impl<T: Config> StakingConfiguration for Pallet<T> {
+		type AssetId = AssetIdOf<T>;
+		type AccountId = AccountIdOf<T>;
+
+		fn configure(
+			asset: Self::AssetId,
+			duration_presets: BTreeMap<DurationSeconds, Perbill>,
+			reward_assets: BTreeSet<Self::AssetId>,
+			early_unstake_penalty: Penalty<Self::AccountId>,
+		) -> Result<(), DispatchError> {
```

Comment:
may manage having some config and consistent updates to it

Reply:
Could you clarify what you mean? Is there any update to make here?